AdobeStockAsset/etc/db_schema.xml

```diff
@@ -0,0 +1,109 @@
+        <column xsi:type="text" name="refresh_token" nullable="true" comment="Refresh Token"/>
+        <column xsi:type="timestamp" name="created_at" on_update="false" nullable="true" comment="Created At"/>
+        <column xsi:type="timestamp" name="updated_at" on_update="false" nullable="true" comment="Updated At"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="id"/>
+        </constraint>
+        <constraint xsi:type="foreign" referenceId="ADO_STO_USR_PRO_ADM_USER_ID" table="adobe_stock_user_profile" column="admin_user_id" referenceTable="admin_user" referenceColumn="user_id" onDelete="CASCADE"/>
+    </table>
+    <table name="adobe_stock_asset" resource="default" engine="innodb" comment="Adobe Stock Asset">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="int" name="media_type_id" padding="10" unsigned="true" nullable="false" identity="false" comment="Media Type ID"/>
+        <column xsi:type="int" name="category_id" padding="10" unsigned="true" nullable="false" identity="false" comment="Category ID"/>
+        <column xsi:type="int" name="creator_id" padding="10" unsigned="true" nullable="false" identity="false" comment="Creator ID"/>
+        <column xsi:type="int" name="premium_level_id" padding="10" unsigned="true" nullable="false" identity="false" comment="Premium Level ID"/>
+        <column xsi:type="int" name="path" padding="10" unsigned="true" nullable="false" identity="false" comment="Path"/>
+        <column xsi:type="int" name="adobe_id" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Adobe Id"/>
+        <column xsi:type="text" name="stock_id" nullable="true" comment="Stock Id"/>
+        <column xsi:type="int" name="is_licensed" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Is Licensed"/>
+        <column xsi:type="text" name="title" nullable="true" comment="Title"/>
+        <column xsi:type="text" name="preview_url" nullable="true" comment="Preview Url"/>
+        <column xsi:type="int" name="preview_width" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Preview With"/>
+        <column xsi:type="int" name="preview_height" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Preview Height"/>
+        <column xsi:type="int" name="width" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Width"/>
+        <column xsi:type="int" name="height" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Height"/>
+        <column xsi:type="int" name="url" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Url"/>
+        <column xsi:type="text" name="country_name" nullable="true" comment="Country Name"/>
```

Comment:
these probably don't have to be nullable true because vector_type is string|null and these just string